Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -449,14 +503,38 @@ const ScrollView = React.createClass({
       };
     }
 
+    const hasStickyHeaders = this.props.stickyHeaderIndices && this.props.stickyHeaderIndices.length > 0;
+
+    const children = hasStickyHeaders ?
+      React.Children.toArray(this.props.children).map((child, index) => {
```

Comment:
Why not React.Children.map?

https://facebook.github.io/react/docs/react-api.html#react.children.map